src/apps/interlink/selftest.snabb

```diff
@@ -0,0 +1,33 @@
+#!snabb snsh
```

Comment:
Does it make sense to run via `#!../../snabb` ?  That is what the `intel_mp` tests do, FWIW.

Reply:
I guess it depends on how you intend run this. In this case `make test` will execute `apps/interlink/selftest.snabb` from `src/`, and that’s also where I tend to run the script from.

@petebristow champions this convention if I understood him correctly and I lean towards following suit. Since snabb is rarely in `PATH`, the next best guess seems to be that its in `PWD`. Of course you can always do `snabb snsh <script>.snabb`.

Reply:
Surely PWD is not in your $PATH though, right?

Reply:
IIRC prior to this change the line was `#!/usr/bin/env snabb snsh` which consults `PATH` but that doesn’t help much usually. On the other hand `#!snabb snsh` won’t consult `PATH` but execute `./snabb`.